filebeat/prospector/prospector_stdin.go

```diff
@@ -8,16 +8,15 @@ import (
 )
 
 type ProspectorStdin struct {
-	Prospector *Prospector
-	harvester  *harvester.Harvester
-	started    bool
+	harvester *harvester.Harvester
+	started   bool
```

Comment:
why we need 'started'? How often do we expect someone to call Run in ProspectorStdin?